apps/bc_desktop/form.yml

```diff
@@ -1,6 +1,13 @@
 ---
 attributes:
-  desktop: "mate"
+  desktop:
+    label: "Desktop Template"
```

Comment:
I don't know if `Template` here makes sense. Remember this is what the customer is going to see.

They want a Gnome desktop - what's a template?

Reply:
Thanks @johrstrom I've changed it to "Desktop Environment"